### ISSUE TITLE
Conflicting rules `Style/LambdaCall` and `Style/MethodCallParentheses` for `.call` method without args

### ISSUE THREAD

**Issue by mockdeep** [bbatsov/rubocop#2569]
When I have a `.call` method that takes no arguments, I can't win between 2 rubocop rules. If I do it this way:

```ruby
thing.call
```

It complains on the ` Style/LambdaCall` rule, and this way:

```ruby
thing.()
```

It complains about ` Style/MethodCallParentheses`. Unfortunately, what rubocop wants in this instance isn't valid ruby syntax:

```ruby
thing.
```

**Comment by alexdowad** [comment]
I have a fix for this. Waiting for my open PR to be merged, then I'll open another one.
